pages/admin/index.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import ApplicationsList from 'components/ApplicationsList/ApplicationsList';
+
+const AdminPage = ({ name }) => {
+  const [applications, setApplications] = useState();
+  useEffect(() => {
+    async function fetchData() {
+      const { data } = await axios.get('/api/applications');
+      setApplications(data.applications);
+    }
+    fetchData();
+  }, []);
+  return (
+    <>
+      <h1>Hello {name}</h1>
+      {applications && <ApplicationsList applications={applications} />}
+    </>
+  );
+};
+
+export async function getServerSideProps({ req, res }) {
+  try {
+    const { data } = await axios.get(
+      `${process.env.HACKNERY_AUTH_URL}/check_token`,
+      {
+        headers: req ? { cookie: req.headers.cookie } : undefined
+      }
+    );
+    if (!data.error) {
+      return { props: data };
+    }
+  } catch (e) {
+    console.error(e.message);
+  }
+  res.writeHead(301, {
```

Comment:
If https://github.com/LBHackney-IT/discretionary-business-grants/pull/121 is merged before this...

```suggestion
  res.writeHead(HttpStatus.MOVED_PERMANENTLY, {
```

This would also need `import * as HttpStatus from 'http-status-codes';` at the top of the file.

...otherwise we can do this in the other PR.

Reply:
I'll address this in the next interaction 👍

Reply:
I just did it in my PR